### DISCUSSION TITLE
Question about Route generated in imported map

### DISCUSSION THREAD

**Discussion by Xialinxuan** [carla-simulator/carla#4814] [Q&A]
I create a simple map in RoadRunner and the OpenDrive output preview is shown as follow.
![rr](https://user-images.githubusercontent.com/39025197/139244668-9855804a-1c70-474f-a5aa-e0f9a1d19366.png)
But when I import this map into Carla, some routes disappear. Routes of the junction at the bottom is like the figure below.
![screenshot](https://user-images.githubusercontent.com/39025197/139246507-6c5b1165-500f-493b-b5e0-4128d5c7fa41.jpg)
Is it because some roads are incomplete in the direction of incoming vehicles? Is there any easy way to create these missing routes?

**Comment by SExpert12** [comment]
Hi,
Can you please tell me how you have used RoadRunner as I also want almost same scenario like yours for my work.

**Reply by Xialinxuan** [reply]
Since I haven't used Roadrunner again after that, I have forgotten the related operations. However, I remember that the design of this scenario should be very simple. After watching the tutorial videos provided by the official website to understand how to use the corresponding tools, you should be able to design such a scenario.

**Reply by SExpert12** [reply]
Okay. Thanks for your reply.
